bin/all_sky_search/pycbc_template_kde_calc

```diff
@@ -4,13 +4,20 @@ import numpy, h5py, operator, argparse, logging
 from pycbc import init_logging
 import pycbc.conversions as convert
 from pycbc import libutils
+from pycbc.events import triggers
 akde = libutils.import_optional('awkde')
 kf = libutils.import_optional('sklearn.model_selection')
 
 parser = argparse.ArgumentParser(description=__doc__)
 parser.add_argument('--signal-file')
 parser.add_argument('--template-file', required=True)
-parser.add_argument('--min-mass', type=float, default=None, required=True)
+parser.add_argument('--min-mass', type=float, default=None,
```

Comment:
This help message does not actually describe what the option does - i.e. remove all signal events with mass2 < min_mass.